pkg/antctl/raw/proxy/command.go

```diff
@@ -147,11 +148,14 @@ func init() {
 	Command.Flags().BoolVar(&o.disableFilter, "disable-filter", false, "If true, disable request filtering in the proxy. This is dangerous, and can leave you vulnerable to XSRF attacks, when used with an accessible port.")
 	Command.Flags().StringVarP(&o.unixSocket, "unix-socket", "u", "", "Unix socket on which to run the proxy.")
 	Command.Flags().DurationVar(&o.keepalive, "keepalive", 0, "keepalive specifies the keep-alive period for an active network connection. Set to 0 to disable keepalive.")
+
```

Comment:
Remove extra empty line?

Reply:
It was on purpose, to separate flags taken from `kubectl proxy` from flags specific to Antrea. I added a comment to make things clearer.